### DISCUSSION TITLE
ditterence between Greaseweazle 4.0 and4.1

### DISCUSSION THREAD

**Discussion by jjcth** [keirf/greaseweazle#399] [Q&A]
I do have both greaseweazles. Same firmware. 4.1 works with PC floppydrive cable etc. Latest tools and Gui. The exact same configuratiuon and sttings do not work with version 4.0. What could be the cause?

**Comment by keirf** [comment]
Why would you use 4.0 anyway?

**Reply by keirf** [reply]
Sorry I'm thinking about FlashFloppy firmware versions for some reason... Ignore that comment!

**Reply by keirf** [reply]
4.0 and 4.1 boards should behave identically. If the 4.0 board isn't working, perhaps it is broken? Anyway in short there are under the hood basically no differences between 4.0 and 4.1 boards.

**Reply by jjcth** [reply]
Hello, I did receive your comments via email and answered those. I am not sure that you received them. In the meantime i have solved the problem with help of your comments. I took away the protective layer on the backside of the board and cleaned the floppy cable contacts. Now it works. I do not know which of the two brought the result but now it works. The board is OK.
Thanks again.

**Reply by keirf** [reply]
I don't think I did. Anyway, glad you got it working!

**Comment by eberhab** [comment]
Hi @keirf, I was also wondering about the differences to 4.0 and features/ fixes that were addressed with v4.1. It's good to know that from a user perspective it should behave the same. For the more curious fans, would you share some more details about what changed internally? Thanks :)

**Reply by keirf** [reply]
Very little changed that isn't obvious. There are a few more USB and power input protections (ESD, current limiting, etc). Apart from that it's just the form factor and more modern USB connector.

**Reply by eberhab** [reply]
Thanks for the great work. It makes our lives easier and more fun :)
